Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/AssetFolderTableView.cpp

```diff
@@ -31,16 +31,44 @@ namespace AzQtComponents
         if (index != rootIndex())
         {
             QAbstractItemView::setRootIndex(index);
-            emit rootIndexChanged(index);
+            emit tableRootIndexChanged(index);
         }
     }
 
-
     void AssetFolderTableView::SetShowSearchResultsMode(bool searchMode)
     {
         m_showSearchResultsMode = searchMode;
     }
 
+    void AssetFolderTableView::mousePressEvent(QMouseEvent* event)
+    {
+        const auto p = event->pos();
+        if (auto idx = indexAt(p); !idx.isValid())
+        {
+            selectionModel()->clear();
+            return;
+        }
+        TableView::mousePressEvent(event);
+    }
+
+    void AssetFolderTableView::mouseDoubleClickEvent(QMouseEvent* event)
+    {
+        const auto p = event->pos();
+        if (auto idx = indexAt(p); idx.isValid())
+        {
+            selectionModel()->select(idx, QItemSelectionModel::SelectionFlag::ClearAndSelect);
+            emit doubleClicked(idx);
+            return;
```

Comment:
don't need this...

Reply:
Well the default double click behavior for this table view tries to edit the asset in whatever column was clicked on, which is why I'm overriding it and just emitting the double click signal

Reply:
sorry, I meant, don't need this return statement which is at the bottom of the function.